packages/datagrid/src/components/datagrid.component.js

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
```

Comment:
This is a component so move into a folder with camel case:
`src/components/DataGrid/DataGrid.component.js` same for the tests